### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by pvlakshm**
Added a note calling out that this template is only to file issues on vstest CLI.

## Description
Please add a meaningful description for this change.
Ensure the PR has required unit tests.

## Related issue
Kindly link any related issues. E.g. Fixes #xyz.

### CODE REVIEW

**Review by smadala — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ## Description
-> Describe the issue you've observed
+> **IMPORTANT**: if the defect is reproduced only in a workflow from within the Visual Studio IDE then do not report the issue here - instead, please report it using Visual Studio's "Send Feedback" option that can be accessed from the Help menu.
```
Comment (codito) [inline .github/ISSUE_TEMPLATE.md]: May be add a link to https://developercommunity.visualstudio.com/spaces/8/index.html

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+>
+> For a defect reproducable from the vstest commanline, describe the issue you've observed.
```
Comment (codito) [inline .github/ISSUE_TEMPLATE.md]: nit: spelling commandline

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 
 ## Steps to reproduce
 > What steps can reproduce the defect?
 > Please share the setup, commandline for vstest.console, sample project, target
 > framework etc.
+>
```
Comment (codito) [inline .github/ISSUE_TEMPLATE.md]: nit: required?

**Review by codito — APPROVED** [review]

**Comment by smadala** [comment]
@dotnet-bot test Windows_NT / Release Build please.

**Pull request merged**